modules/stochastic_tools/include/samplers/AdaptiveImportanceSampler.h

```diff
@@ -85,6 +85,9 @@ class AdaptiveImportanceSampler : public Sampler, public ReporterInterface
   /// True if the sampling is completed
   bool _is_sampling_completed;
 
+  /// Flag samples if the surrogate prediction was inadequate
+  // const bool & _flag_sample;
+
```

Comment:
Delete this. BTW I was wondering something similar. Can you create one AIS object that can either use flag_sample for AL or not for regular behavior?

Reply:
Yes, technically. flag_sample is used to facilitate parallelization. This version of AIS uses one Markov chain that has to be run in serial. But if there is desire to use multiple Markov chains for AIS, then flag_sample may be used.